.gitignore

```
@@ -49,3 +49,6 @@ tsconfig.tsbuildinfo
 
 # rollup cache
 .rollup.cache/
+
+# build output
+build/
```

Add a newline at the end of the file.
```suggestion
build/
```